packages/rum-core/src/rumEvent.types.ts

```diff
@@ -14,7 +14,7 @@ export type RumActionEvent = CommonProperties & {
   /**
    * RUM event type
    */
-  readonly type: string
+  readonly type: 'action'
```

Comment:
This is not an issue, but I am just curious how did this type change from `string` to `'action'`? did we change the something in the tool that generates the types?

Reply:
Yes! we updated our `json-schema-to-typescript` fork with the  latest changes from upstream some time ago: https://github.com/bcaudan/json-schema-to-typescript/pull/1